azkaban/remote.py

```diff
@@ -737,6 +753,37 @@ def get_workflow_info(self, name, flow):
         # but sends a 200 empty response if the project doesn't exist
         raise AzkabanError('Project %s not found.', name)
 
+  def get_workflow_graph(self, name, flow):
+    """Get graph of jobs corresponding to a workflow.
+
+    :param name: Project name.
+    :param flow: Name of flow in project.
+
+    """
+    self._logger.debug(
+      'Fetching flow graph for workflow %s in project %s', flow, name
+    )
+    try:
+      res = self._request(
+        method='GET',
+        endpoint='manager',
+        params={
+          'ajax': 'fetchflowgraph',
+          'project': name,
+          'flow': flow,
+        },
+      )
+    except HTTPError:
```

Comment:
The logic here is the same as in `get_workflow_info`; do you mind extracting it to a helper function (say `_extract_workflow_json`)?